app/api/business/brief_overview_business.py

```diff
@@ -74,6 +79,53 @@ def close_opportunity_early(current_user, brief_id):
     return brief
 
 
+def withdraw_opportunity(current_user, brief_id, data):
+    brief = brief_service.get(brief_id)
+    if not brief:
+        raise NotFoundError('Opportunity {} does not exist'.format(brief_id))
+
+    if not brief_service.has_permission_to_brief(current_user.id, brief_id):
+        raise UnauthorisedError('Not authorised to withdraw opportunity {}'.format(brief_id))
+
+    if brief.status != 'live':
+        raise BriefError('Unable to withdraw opportunity {}'.format(brief_id))
+
+    brief = brief_service.withdraw_opportunity(brief, data['reasonToWithdraw'])
+
+    user = users.get(current_user.id)
+    if not user:
+        raise NotFoundError('User {} does not exist'.format(current_user.id))
```

Comment:
Should the `current_user` argument passed in to this function be `user_id` instead? if `current_user` is a user instance already, then presumably this check would have occurred before calling this function.

Reply:
I've passed in the user's ID as suggested.